### PR TITLE
[ConsumptionMgmt] Adding two new operations for Reservations

### PR DESCRIPTION
**Pull request by kc786**
1. Reservation Summaries
2. Reservation Details

### CODE REVIEW

File: azure-mgmt-consumption/HISTORY.rst [hunk 1]
```diff
@@ -3,19 +3,7 @@
...
-
-- Featuring stable api GA version 2017-11-30
-- Supporting EA customers with azure consumption usage details
-
-**Breaking changes**
-
-- Removing support for calling usage_details.list() with 'invoice_id'. Will feature in future releases.
-
-0.1.0 (2017-05-18)
+0.1.0 (1970-01-01)
```
Comment (lmazuel) [inline azure-mgmt-consumption/HISTORY.rst]: Please do not override the ChangeLog, and even provide a 1.1.0 explaination of the reservations change

File: azure-mgmt-consumption/README.rst [hunk 2]
```diff
@@ -37,8 +37,8 @@ Usage
 =====
 
 For code examples, see `Consumption Management
-<https://docs.microsoft.com/python/api/overview/azure/consumption>`__
-on docs.microsoft.com.
+<https://azure-sdk-for-python.readthedocs.org/en/latest/sample_azure-mgmt-consumption.html>`__
```
Comment (lmazuel) [inline azure-mgmt-consumption/README.rst]: Please restore previous link

File: azure-mgmt-consumption/tests/test_mgmt_consumption.py [hunk 3]
```diff
@@ -12,6 +12,9 @@
 from devtools_testutils import AzureMgmtTestCase, ResourceGroupPreparer
 
 class MgmtConsumptionTest(AzureMgmtTestCase):
+    
+    reservationOrderId = 'ca69259e-bd4f-45c3-bf28-3f353f9cce9b'
```
Comment (lmazuel) [inline azure-mgmt-consumption/tests/test_mgmt_consumption.py]: Please confirm none of these values are confidentials

Comment (kc786) [reply azure-mgmt-consumption/tests/test_mgmt_consumption.py]: Those reservations belong to the test account that we use

File: azure-mgmt-consumption/HISTORY.rst (hunk 1 above)
Comment (kc786) [reply azure-mgmt-consumption/HISTORY.rst]: I believe i did not make any changes manually to this file. t happens when i ran 
cookiecutter.exe -f gh:Azure/cookiecutter-azuresdk-pypackage
before creating pull request as mentioned here . 
https://github.com/Azure/azure-sdk-for-python/wiki/Generate-a-Python-SDK-from-a-Swagger-file
Am i missing something here

**Pull request merged**